.gitignore

```diff
@@ -56,3 +56,6 @@ docs/_build/
 
 # PyBuilder
 target/
+
+**/Manifest.toml
```

Comment:
Yes it does. But, more than just Manifest.toml and Project.toml.  It optionally puts a depo in `./depot` in the `diffeqpy` directory. I also don't like it, but this was easier to start with. I agree it should be changed. Probably under `~/.julia/julia_project`.  Both `juliaup` and `pythoncall` claim directories there.

Reply:
I think it's very rare that you'd need a separate depot. If a Julia environment is enough, I think that's much better. It definitely helps reduce precompile time. It's different from how Python virtualenv/venv works but I think a slug-based mapping is enough for bridging the gap.

Reply:
I'd rather just use a Julia environment as well. In fact, I do use one. But the separate depot is just to work around the problem of incompatible libpython. I want to avoid having multiple python projects fighting over PyCall, always rebuilding it....  Or maybe I don't follow what you mean here.

Reply:
I don't have a great understanding of the problem with incompatible libpython. My assumption, which I am not sure of, is that it is not enough to have a separate `.ji` file for `PyCall` for each python project. Some other `.ji` files might need to be recompiled and be incompatible as well. Or, is it enough to use multiple depots and one of them only contains `PyCall` (including the compiled `.ji` file) ?

Reply:
> But the separate depot is just to work around the problem of incompatible libpython.

I'm pretty sure the direction of https://github.com/JuliaPy/PyCall.jl/pull/945 makes this hack unnecessary. It let us configure libpython for each Julia project and so PyCall.jl can be precompiled for each libpython. It'll be a game changer for how Julia packages are used from Python.

Reply:
Wow. I looked around quite a bit, but missed this. It's a big deal. I'll have to do some reading to absorb it all.